tests/Mapping/StaticPHPDriverTest.php

```diff
@@ -15,24 +17,23 @@ public function testLoadMetadata(): void
         $metadata = $this->createMock(ClassMetadata::class);
         $metadata->expects(self::once())->method('getFieldNames');
 
-        $driver = new StaticPHPDriver([__DIR__]);
-        $driver->loadMetadataForClass(TestEntity::class, $metadata);
+        $driver = new StaticPHPDriver([]);
+        $driver->loadMetadataForClass(Entity::class, $metadata);
     }
 
     public function testGetAllClassNames(): void
     {
-        $driver     = new StaticPHPDriver([__DIR__]);
```

Comment:
This directory contains `Doctrine.Tests.Persistence.Mapping.PHPTestEntityAssert.php` that cannot be includes without setting the `$metadata` variable. Not supported by the `StaticPHPDriver`.